packages/ui/src/proposals/components/ProposalList/ProposalList.tsx

```diff
@@ -13,18 +14,21 @@ export interface ProposalListProps {
   isPast?: boolean
 }
 
-export const ProposalList = ({ proposals, isPast }: ProposalListProps) => (
-  <RowGapBlock gap={4}>
-    <ProposalsListHeaders $colLayout={ProposalColLayout}>
-      <ListHeader />
-      <ListHeader>Stage</ListHeader>
-      <ListHeader>Proposer</ListHeader>
-      <ListHeader />
-    </ProposalsListHeaders>
-    <List as="div">
-      {proposals.map((proposal) => (
-        <ProposalListItem key={proposal.id} proposal={proposal} isPast={isPast} />
-      ))}
-    </List>
-  </RowGapBlock>
-)
+export const ProposalList = ({ proposals, isPast }: ProposalListProps) =>
+  proposals.length ? (
```

Comment:
I'd avoid big ternary operator blocks. An `if` will add some visual space, which I find easier to read:

```tsx
() => {
  if (!proposals.length) {
    return <NotFoundText>No.... </NotFoundText>
  }
  
  return (
    <RowGapBlock>
      //...
    </RowGapBlock>
  )
}
```